addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -350,6 +356,14 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
         });
     },
 
+    _displaySessionClosedPage:function () {
+        this.$('.o_survey_question_header').addClass('invisible');
+        this.$('.o_survey_session_results').addClass('d-none');
+        this.$('.o_survey_session_description_done').prepend($("<h1>").text(_t('Thank you!'))).removeClass('d-none');
+        this.$('.o_survey_session_navigation_previous').addClass('d-none');
+        this.$('.o_survey_session_navigation_next').addClass('d-none');
```

Comment:
Thanks for the proposition but I think the template is already hard to read and I don't think it could simplify other place as the only place we deal with the session closed is there (other place hide some stuff but during the session and not all at once).
So to reduce the number of line, I have used a for each and "invisible" everywhere (instead of d-none and invisible). What do you think ?

Reply:
ping @pyduf :)

Reply:
@flch-odoo: Yes, it is better that way; I didn't know that syntax, thanks.